apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -465,13 +477,16 @@ def render(element_html: str, data: pl.QuestionData) -> str:
 
         for option in student_previous_submission:
             submission_indent = option.get("indent", None)
+
             if submission_indent is not None:
+                if block_orientation is BlockOrientationType.HORIZONTAL:
+                    raise Exception(
+                        'When block-orientation is "horizontal" indentaion is not supported. Please remove "indent" attributes from all pl-answer blocks.'
+                    )
```

Comment:
I don't think this is needed, don't we already have this case covered?

Having an `indent` attribute on a block is already disallowed unless indentation=true, but you are already disallowing that below.

Reply:
Ah! Didn't catch that. It is now removed.